app/test_main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from unittest import mock
+import pytest
```

Comment:
`pytest` is a 3rd party test framework and `unittest` is a built-in python test framework, which is why you should separate them with one blank line as well. Take this into account in the future.